apollo/server/resolvers/person.ts

```diff
@@ -209,6 +209,38 @@ const updatePersonResolver: ResolverFn<
   }
 };
 
+/**
+ * The resolver for importPeople Mutation
+ * @async
+ * @param arg - The arguments for the importPeople mutation
+ * @returns The update response
+ * @category Mutation Resolver
+ */
+const importPeopleResolver: ResolverFn<
+  { people: PersonCreationAttributes[] },
+  PersonUpdateResponse[]
+> = async (
+  _,
+  { people },
+  { user, dataSources }
+): Promise<PersonUpdateResponse[]> => {
+  const isAdmin = Boolean(
+    user && (await dataSources.executiveAPI.findExecutive(user.sid))
+  );
+  if (!isAdmin) {
+    return [{ success: false, message: "You have no permission to do this" }];
+  }
+  const result = people.map(async (person) => {
```

Comment:
should be `people.map((person) => async () => {...})`

Reply:
better call this `mutations` not `result`?

Reply:
better call result_1, result_2, ..., result_n

Reply:
you on9?